pegasus/sites.v3/code.org/public/css/brstrap.css

```diff
@@ -62,8 +68,44 @@
   .mobile-show-medium { display: none }
 
   .i-agree-button { width: 9%; }
-}
 
+ /*This is applicable when side by side images with rounded corners need to be stacked on smaller sizes (e.g. code.org/student)*/
```

Comment:
Do you think I should include the different media screen sizes in the css file for a specific page if it has unique responsive elements not handled by brstrap, such as these corners?